spec/requests/admin_tasks_spec.rb

```diff
@@ -81,6 +81,16 @@
             end
           end
 
+          context "when a task's passed flag is not set" do
+            it "doesn't create a task and shows an error" do
+              expect {
+                post admin_claim_tasks_path(claim, name: "qualifications")
+              }.not_to change { Task.count }
```

Comment:
I know all the other specs use `Task.count`, but I think we can be more specific here:
```suggestion
              }.not_to change { claim.tasks.count }
```